jetty-core/jetty-compression/jetty-compression-server/src/main/java/org/eclipse/jetty/compression/server/internal/CompressionResponse.java

```diff
@@ -60,8 +60,13 @@ public void write(boolean last, ByteBuffer content, Callback callback)
                 int status = getStatus();
                 if (status > 0 && (
                     HttpStatus.isInformational(status) ||
+                    // We don't use HttpStatus.hasNoBody(status) here
+                    // because we need to handle 304 Not Modified differently for etags
                     status == HttpStatus.NO_CONTENT_204 ||
-                    status == HttpStatus.RESET_CONTENT_205) &&
+                    status == HttpStatus.RESET_CONTENT_205 ||
+                    status == HttpStatus.PARTIAL_CONTENT_206 ||
```

Comment:
206 responses can be compressed

Reply:
Note, Chrome/Chromium doesn't even support compressed content on a 206 response anyway.

https://issues.chromium.org/issues/423403084

Reply:
There's also the technical details on the user-agent side, how do you decompression partial content?
As most compression algorithms want some context for later bytes.